src/server.c

```diff
@@ -693,22 +693,37 @@ int allPersistenceDisabled(void) {
 
 /* ======================= Cron: called every 100 ms ======================== */
 
-/* Add a sample to the operations per second array of samples. */
-void trackInstantaneousMetric(int metric, long long current_reading) {
-    long long now = mstime();
-    long long t = now - server.inst_metric[metric].last_sample_time;
-    long long ops = current_reading -
-                    server.inst_metric[metric].last_sample_count;
-    long long ops_sec;
-
-    ops_sec = t > 0 ? (ops*1000/t) : 0;
-
-    server.inst_metric[metric].samples[server.inst_metric[metric].idx] =
-        ops_sec;
+/* Add a sample to the operations per second array of samples. This takes total operation count and
+ * current time, then calculates and records the operation rate between to samples. */
+void trackInstantaneousRateMetric(int metric, long long current_reading, long long current_time) {
+    if (server.inst_metric[metric].last_sample_time > 0) {
+        long long t = current_time - server.inst_metric[metric].last_sample_time;
```

Comment:
Looks like this is now a duration.  Let's name it "duration" rather than "t".  Is this us? ms?